### PR TITLE
Berserker overhaul / Rising bass added.

### PR DESCRIPTION
**Pull request by RinPin**
## About The Pull Request

Hehehe ohoho funny unarmed changes - 
TLDR with the advent of melee becoming stronger under recent changes, and the ability for backstabbing, as well as sleeping carp now being compatible with guns, i made this PR to give unarmed some more well defined roles in the form of offense with berserker and  and defense  with rising bass.

Changes - Berserker has some damage increases that are below the original ones, but still equivalent to what a melee weapon would get with backstabs when the correct combos are performed, it also received a no slip trait to really encourage the aggressive nature, as the oil that seems to be everywhere on the floor really punishes melee players when they're aggressive.

Rising bass was now added to the round start supply kit, as a more defensive focused alternative to berserker, lacking any direct simple damage boosts but instead offering projectile dodges and some extra defensive moves that work on players and mobs alike, it still retains the no drugs / no guns trait unlike carp.

## Pre-Merge Checklist
- [ Y] You tested this on a local server.
- [ Y] This code did not runtime during testing.
- [ Y] You documented all of your changes.

## Changelog
:cl:
add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

**Review by Kilmented — APPROVED** [review]
hope I don't have to revert the Bass thing
